core/src/main/java/org/elasticsearch/index/query/QueryParseContext.java

```diff
@@ -19,16 +19,16 @@
 
 package org.elasticsearch.index.query;
 
-import java.io.IOException;
-import java.util.Objects;
-
 import org.elasticsearch.common.ParseField;
 import org.elasticsearch.common.ParseFieldMatcher;
 import org.elasticsearch.common.ParseFieldMatcherSupplier;
 import org.elasticsearch.common.ParsingException;
 import org.elasticsearch.common.xcontent.XContentParser;
 import org.elasticsearch.indices.query.IndicesQueriesRegistry;
 
+import java.io.IOException;
+import java.util.Objects;
```

Comment:
funny how they keep on bouncing back and forth. who's right between @cbuescher 's ide and mine? :)